### PR TITLE
Added kaspactl command to request balance by wallet address

### PR DESCRIPTION
**Pull request by FestinaLente666**

### CODE REVIEW

File: app/appmessage/message.go [hunk 1]
```diff
@@ -124,6 +124,8 @@ const (
 	CmdStopNotifyingUTXOsChangedResponseMessage
 	CmdGetUTXOsByAddressesRequestMessage
 	CmdGetUTXOsByAddressesResponseMessage
+	CmdGetBalanceByAddressRequest
```
Comment (someone235) [inline app/appmessage/message.go:127]: Please add `Message` suffix

File: app/appmessage/rpc_get_balance_by_address.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+	Error *RPCError
+}
+
+// Command returns the protocol command string for the message
+func (msg *GetBalanceByAddressResponse) Command() MessageCommand {
+	return CmdGetBalanceByAddressResponse
+}
+
+// NewGetBalanceByAddressResponse returns a instance of the message
```
Comment (someone235) [inline app/appmessage/rpc_get_balance_by_address.go:36]: a -> an

File: app/rpc/rpchandlers/get_balance_by_address.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		errorMessage.Error = appmessage.RPCErrorf("Method unavailable when kaspad is run without --utxoindex")
+		return errorMessage, nil
+	}
+
+	getBalanceByAddressRequest := request.(*appmessage.GetBalanceByAddressRequest)
+
+	var balance uint64 = 0
+	addressString := getBalanceByAddressRequest.Address
+
+	address, err := util.DecodeAddress(addressString, context.Config.ActiveNetParams.Prefix)
```
Comment (someone235) [inline app/rpc/rpchandlers/get_balance_by_address.go:24]: error is unhandled

File: infrastructure/network/netadapter/server/grpcserver/protowire/README.md [hunk 4]
```diff
@@ -3,13 +3,13 @@ protowire
 
 1. Download and place in your PATH:
    https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
-2. `go get github.com/golang/protobuf/protoc-gen-go`
-3. `go get google.golang.org/grpc/cmd/protoc-gen-go-grpc`
+2. `go install github.com/golang/protobuf/protoc-gen-go`
```
Comment (someone235) [inline infrastructure/network/netadapter/server/grpcserver/protowire/README.md:6]: Why did you change the readme?

**Review by someone235 — CHANGES_REQUESTED** [review]

File: infrastructure/network/netadapter/server/grpcserver/protowire/README.md (hunk 4 above)
Comment (FestinaLente666) [reply infrastructure/network/netadapter/server/grpcserver/protowire/README.md:6]: Because what's currently written there is deprecated and no longer works
